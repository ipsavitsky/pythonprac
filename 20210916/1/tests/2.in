"2 + 3*2", "445"+"221", 1.5
