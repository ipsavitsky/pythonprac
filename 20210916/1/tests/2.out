<class 'str'> <class 'str'> <class 'float'>
